app/controllers/buyers/accounts/bulk/send_emails_controller.rb

```diff
@@ -1,18 +1,24 @@
+# frozen_string_literal: true
+
 class Buyers::Accounts::Bulk::SendEmailsController < Buyers::Accounts::Bulk::BaseController
 
-  def new
-  end
+  def new; end
 
   def create
     @errors = []
 
     @accounts.each do |recipient|
-      message = current_account.messages.build params[:send_emails]
+      message = current_account.messages.build send_emails_params
       message.to = recipient
       @errors << message unless message.save && message.deliver!
     end
 
     handle_errors
   end
 
+  private
+
+  def send_emails_params
+    params.fetch(:send_emails).permit!
```

Comment:
On the one hand it worked seemingly without protection in the past. On the other hand, perhaps we need to filter out some parameters from message. Maybe check docs.

Reply:
I tend to agree with you to filter but it is also OK to do the filtering in another PR later ;) In the latter case, open an issue to not forget about it and schedule it

This will really pose an issue when we remove `protected_attributes` gem

Reply:
Probably we need to revisit all `#permit!` usages.

Reply:
I added a line to https://issues.redhat.com/browse/THREESCALE-5010

Reply:
I updated all params usages and hopefully now it's done correctly.